pom.xml

```diff
@@ -39,7 +39,8 @@
     <properties>
         <!-- Primary dependencies - maintained manually -->
         <camel.major.minor>3.11</camel.major.minor> <!-- run after each change: cd docs && mvnd validate -->
-        <camel.version>${camel.major.minor}.1</camel.version>
+        <camel-community.version>${camel.major.minor}.1</camel-community.version>
+        <camel.version>3.11.1.fuse-800009</camel.version>
```

Comment:
Should we align the `camel-dependencies` parent version with this?

Reply:
Good catch!

Reply:
If we use the productized version (3.11.1.fuse-800009-redhat-00001) for camel.version it won't disappear in two weeks like the dev build will.

Reply:
OK, let me use that one.